src/argilla/server/models.py

```diff
@@ -34,26 +34,54 @@ def default_inserted_at(context):
     return context.get_current_parameters()["inserted_at"]
 
 
+class AnnotationType(str, Enum):
+    text = "text"
+    single_label = "single_label"
+
+
 class UserRole(str, Enum):
     admin = "admin"
     annotator = "annotator"
 
 
+class Annotation(Base):
+    __tablename__ = "annotations"
+
+    id: Mapped[UUID] = mapped_column(primary_key=True, default=uuid4)
+    name: Mapped[str]
+    title: Mapped[str] = mapped_column(Text)
+    type: Mapped[Optional[AnnotationType]] = mapped_column(default=AnnotationType.text)
```

Comment:
Why this attribute is optional?

Reply:
We have a default `AnnotationType.text`, that's why it's optional. If you don't specify one then we are using the `text` annotation type.

Reply:
I have been rechecking and with a `default` value the `Optional` is useless. We can remove it.